tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -536,23 +538,23 @@ def test_whisper_timestamp_prediction(self):
                             "tight-loan cloth that was the only garment he wore, the "
                             "cut"
                         ),
-                        "timestamp": (5.5, 11.94),
+                        "timestamp": (5.5, 11.95),
```

Comment:
@sgugger This should feel less scary ?

Also the audio sent here is 30s long, so the timestamp modifications looks like a fix (although given the unbounded nature of whisper, it's hard to be 100% sure about it it could very well be the model outputting weird stuff)

Reply:
I'll wait for you to be OK with this before merging.

Reply:
Ok I'll merge.